### DISCUSSION TITLE
PS1/PSX BIOS Error, also -Swanstation

### DISCUSSION THREAD

**Discussion by pocket-books** [OnionUI/Onion#486] [Q&A]
Every time I try adding any of the SCPH bios files to any of the three instances of the included PSX core I get a 'no bios found' error in Retroarch.

All BIOS files coppied to both:
MiyooSD_Card/RetroArch/.retroarch/system
MiyooSD_Card/BIOS
- though the file browser on retroarch won't go into the .retroarch folder as it is now.

Would anyone please clue me in to how to add the bios files to the preinstalled core?
scph1001.bin - MD5:  DC2B9BF8DA62EC93E868CFD29F0D067D
sscph5501.bin	- MD5:  490F666E1AFB15B7362B406ED1CEA246
etc. etc. etc. ...

There's also the issue of adding Swanstation to Retroarch and Onion OS by extension.
Any help is appreciated

**Comment by pocket-books** [comment]
Pardon my lack of specifics here
A quick update:
I'd found the PCSX_ReArmed settings file and manually entered the location of the scph1001.bios.
It found the propper bios then but there was no sound and games were running really fast on their own without using fastforward.
Selecting a synch setting locked the game back to a stable frame rate but there was still no sound.

Reinstalling Onion OS from scratch to see if anything more can be done.

**Comment by pocket-books** [comment]
While Swanstation on the Miyoo Mini might be a long shot and using the file explorer in .hidden folders isn't a problem anymore.

- I am still having trouble getting the Playstation core to find the drivers located in /BIOS after a clean instal.

**Comment by zebramonkeylizard** [comment]
> scph1001.bin - MD5: DC2B9BF8DA62EC93E868CFD29F0D067D

Mine: 
SCPH1001.BIN - MD5: 924e392ed05558ffdb115408c263dccf
I recommend using the following though, as it has several benefits, e.g., speed, region free. 
psxonpsp660.bin - MD5: c53ca5908936d412331790f4426c6c33

**Comment by zebramonkeylizard** [comment]
Take a look at the BIOS section here: 
[https://docs.libretro.com/library/pcsx_rearmed/](url)

**Comment by pocket-books** [comment]
I used that bios you listed with the correct MD5 checksum.
What is the best way to assign bios files to cores?
All the bios files are in SD_Card/BIOS
Should OnionOS/Retroarch be finding it automatically?

This is all done from a clean install by the way.

**Reply by zebramonkeylizard** [reply]
I have mine in the SD_Card/BIOS folder and it works. Maybe the filename is the wrong letter case. e.g., scph1001.bin --> SCPH1001.BIN.
